ehrql/tables/beta/tpp.py

```diff
@@ -501,3 +502,24 @@ class open_prompt(EventFrame):
         constraints=[Constraint.NotNull()],
         description="The response to the question",
     )
+
+
+@table
+class apcs_cost(EventFrame):
+    apcs_ident = Series(
+        int,
+        constraints=[Constraint.NotNull()],
+        description="apcs_ident",
```

Comment:
@inglesp where is the best place to find the description for all the new variables? some may be in cohort-extractor repo or on slack? or should I leave the variable name in the description for now as a placeholder?

Reply:
I seem to recall there's a (looong) document published by the NHS that describes all of the SUS fields.

For now, what about a placeholder (probably `"TODO"`) so that users can get on with using this?

Reply:
thx, I couldnt find this document and have changed the description of all fields to TODO for now